### PR TITLE
Convert dialogs to C++

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

**Comment by marijnvdwerf** [comment]
@IntelOrca Is this the approach you meant?

File: src/openrct2/windows/debug_paint.cpp [hunk 1]
```diff
@@ -24,6 +25,16 @@
...
+=======
+
+extern "C" {
+    #include "../interface/widget.h"
+    #include "../localisation/localisation.h"
+    #include "../paint/map_element/map_element.h"
+    #include "../paint/paint.h"
+    #include "../ride/track_paint.h"
+}
+>>>>>>> Stashed changes:src/openrct2/windows/debug_paint.cpp
```
Comment (IntelOrca) [inline src/openrct2/windows/debug_paint.cpp]: You got some git noise here.

**Comment by IntelOrca** [comment]
Yep, just basically do the bare minimum changes to get all the windows to be `.cpp`.

**Comment by Gymnasiast** [comment]
C++ file names should be capitalised. (Some of them aren't, but that is accidental.)

**Comment by wolfreak99** [comment]
When this gets merged, could you tag me? Just so I know to rebase some of my work.

**Comment by janisozaur** [comment]
I only skimmed the changes, but it looks OK to me. I'd still like @IntelOrca to decide if it's ready to merge.

**Pull request merged**